### PR TITLE
7_4_X_AlCaPhiSym_ringCut

### PR DESCRIPTION
**Pull request by bmarzocc**
Possibility to put different ring dependent cut on ADC

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
How should the module be configured to reproduce the old behaviour?

File: HLTrigger/special/src/HLTEcalPhiSymFilter.cc [hunk 1]
```diff
@@ -43,11 +49,19 @@ HLTEcalPhiSymFilter::fillDescriptions(edm::ConfigurationDescriptions& descriptio
   desc.add<edm::InputTag>("endcapHitCollection",edm::InputTag("ecalRecHit","EcalRecHitsEE"));
   desc.add<unsigned int>("statusThreshold",3);
   desc.add<bool>("useRecoFlag",false);
-  desc.add<double>("ampCut_barrel",8.);
-  desc.add<double>("ampCut_endcap",12.);
+  desc.add<std::vector<double> >("ampCut_barrelP",{6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,
```
Comment (Martin-Grunewald) [inline HLTrigger/special/src/HLTEcalPhiSymFilter.cc]: Leave the above two in, but mark them as deprecated, like in this example:
```
desc.addOptional<bool>("verbose", false)->setComment("Deprecated and to be removed");
```
Thia bove is for the 74X PR ONLY!!

fillDescriptions of the newly added parameters must be set so as to reproduce the old behaviour in both the 74X and 75X PR.

**Comment by Martin-Grunewald** [comment]
I presume to reproduce the old cuts, all cuts should either 8.0 (barrel) and 12.0 (endcap), correct?

**Comment by bmarzocc** [comment]
Hi Martin,
In principle PR #9233 and #9232 were done in order to change the way the cuts are applied in the PhiSymmetry stream, in order to have ring dependent cuts. By now for simplicity we use same numbers, but we also want to have the flexibility to change them in the future if needed. Is it still possible to change the path code?

**Comment by Martin-Grunewald** [comment]
The PRs are not accepted yet. Make the changes and update the branch in your git repository which will automatically update the PRs. Of course we can then later change the parameter values.
The defaults must however be set as to reproduce the old behaviour (using 8 and 12).

**Comment by bmarzocc** [comment]
Everything modified as requested. Any news?

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**